cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/CristinFundingAmount.java

```diff
@@ -0,0 +1,53 @@
+package no.unit.nva.cristin.projects.model.cristin;
+
+import static no.unit.nva.cristin.model.JsonPropertyNames.AMOUNT;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import java.util.Objects;
+
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class CristinFundingAmount {
+
+    public static final String CRISTIN_CURRENCY_CODE = "currency_code";
+
+    @JsonProperty(CRISTIN_CURRENCY_CODE)
+    private final transient String currencyCode;
+    @JsonProperty(AMOUNT)
+    private final transient Double amount;
+
+    @JsonCreator
+    public CristinFundingAmount(@JsonProperty(CRISTIN_CURRENCY_CODE) String currencyCode,
+                                @JsonProperty(AMOUNT) Double amount) {
+        this.currencyCode = currencyCode;
+        this.amount = amount;
+    }
+
+    public String getCurrencyCode() {
+        return currencyCode;
```

Comment:
Is the currency code an ISO4217 code?

Reply:
The string "NOK" is hardcoded in the Cristin model. We have to change that when we add support for creation and update of funding amount